src/PermissionRegistrar.php

```diff
@@ -59,7 +59,7 @@ public function forgetCachedPermissions()
 
     public function getPermissions(): Collection
     {
-        return $this->cache->remember($this->cacheKey, config('permission.cache_expiration_time'), function () {
+        return $this->cache->forever($this->cacheKey, function () {
```

Comment:
i removed it because cache gets cleared on each create/update/delete event, so why the extra clearing each 24hrs ?